demo/src/main/java/com/novoda/storagepathfinder/demo/DemoDependenciesFactory.java

```diff
@@ -26,15 +19,6 @@ public static AssetCloner createAssetCloner(Context context) {
     }
 
     public static AndroidDeviceStorageInspector createStorageInspector(Context context) {
-        ExternalStorageDirectories externalStorageDirectories = new AndroidExternalStorageDirectories(context.getApplicationContext());
-        FileSystem fileSystem = new AndroidFileSystem(externalStorageDirectories);
-        DeviceFeatures deviceFeatures = new AndroidDeviceFeatures();
-        AndroidSystem androidSystem = new AndroidSystem();
-        return new AndroidDeviceStorageInspector(
-                context.getApplicationContext(),
-                fileSystem,
-                deviceFeatures,
-                androidSystem
-        );
+        return AndroidDeviceStorageInspector.builder(context).build();
```

Comment:
I'd remove this from the factory now, better to show the real usage of the `builder` where it is called.

Reply:
Good point, done in cbeccdbe34875b2f6b6c8ba267ac1ae1c7c3f791